.drone.yml

```diff
@@ -4721,6 +4721,7 @@ name: build-buildboxes
 environment:
   GID: "1000"
   RUNTIME: go1.17.3
+  BUILDBOX_VERSION: teleport8
```

Comment:
Is the idea here that once this is backported we will then update master to something other than `teleport8`?

Reply:
This is targeted at `branch/v8`, and I've added it to the Drone pipeline so it will build the images on merges to that as well as master. `master` should be untouched.

Reply:
We can forward-port it to `master` with `teleport9` if that makes sense, but I was mainly focussed on unjamming `v8`